models/chainlink/bnb/chainlink_bnb_ocr_fulfilled_transactions.sql

```diff
@@ -0,0 +1,53 @@
+{{
+  config(
+    tags=['dunesql'],
+    alias='ocr_fulfilled_transactions',
+    materialized='view',
+    post_hook='{{ expose_spells(\'["bnb"]\',
+                                "sector",
+                                "chainlink",
+                                \'["linkpool_ryan"]\') }}'
+  )
+}}
+
+{% set incremental_interval = '7' %}
+
+WITH
+  bnb_usd AS (
+    SELECT
+      minute as block_time,
+      price as usd_amount
+    FROM
+      {{ source('prices', 'usd') }} price
+    WHERE
+      symbol = 'BNB'
+      {% if is_incremental() %}
+        AND minute >= date_trunc('day', now() - interval '{{incremental_interval}}' day)
+      {% endif %}      
+  ),
+  ocr_fulfilled_transactions AS (
+    SELECT
+      MAX(tx.block_time) as block_time,
+      tx."from" as "node_address",
+      MAX((cast((gas_used) as double) / 1e18) * gas_price) as token_amount,
+      MAX(bnb_usd.usd_amount) as usd_amount
+    FROM
+      {{ source('bnb', 'transactions') }} tx
+      RIGHT JOIN {{ ref('chainlink_bnb_ocr_gas_transmission_logs') }} ocr_gas_transmission_logs ON ocr_gas_transmission_logs.tx_hash = tx.hash
```

Comment:
we've seen some poor performance on joins like this. you don't notice in CI tests because it's simply building a view, not running the query. you would likely see the poor performance later downstream.

you're essentially joining base transactions to base logs, both massive tables, especially on bnb. it would be good to apply more join conditions as well -- block number? date filters (i.e. incremental block filter on join conditions for both historical run and incremental)?